### ISSUE TITLE
Voice Over doesn't announce whether a folder is expanded or collapsed in file explorer

### ISSUE THREAD

**Issue by mikolysz** [microsoft/vscode#158634]
Type: <b>Bug</b>

# Steps to reproduce:

1. Open an Existing folder in VS Code, preferably containing some code.
2. Turn on Voice Over.
3. Press Command + Shift + e to focus the explorer view.
4. Use the up and down arrow keys to focus a folderr. Voice Over will not announce whether the folder is expanded or collapsed. Exapnd the foldrr, focus it again, and the state is still not announed.

VS Code version: Code 1.70.2 (Universal) (e4503b30fc78200f846c62cf8091b76ff5547662, 2022-08-16T05:36:37.829Z)
OS version: Darwin arm64 21.5.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|4, 3, 3|
|Memory (System)|16.00GB (0.07GB free)|
|Process Argv|--crash-reporter-id 0a0a4d05-10b5-444e-817f-e34805adc4dd|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (19)</summary>

Extension|Author (truncated)|Version
---|---|---
gotemplate|cas|0.4.0
copilot|Git|1.42.6544
go|gol|0.35.2
vscode-docker|ms-|1.22.1
csharp|ms-|1.25.0
python|ms-|2022.12.1
vscode-pylance|ms-|2022.8.30
jupyter|ms-|2022.7.1102252217
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.9
remote-containers|ms-|0.245.1
java|red|1.9.0
indent-nav|Ton|1.0.3
vscodeintellicode|Vis|1.2.22
vscode-java-debug|vsc|0.43.0
vscode-java-dependency|vsc|0.21.0
vscode-java-pack|vsc|0.25.0
vscode-java-test|vsc|0.36.0
vscode-maven|vsc|0.37.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411cf:30542925
vsaa593:30376534
pythonvs932:30410667
wslgetstarted:30449410
cppdebug:30492333
vscaac:30438847
pylanb8912:30545647
vsclangdc:30486549
c4g48928:30535728
d7fjc500:30543455
dsvsc012cf:30540253
azure-dev_surveyone:30548225

```

</details>

**Comment by isidorn** [comment]
@mikolysz thanks for opening this issue. We set all the proper aria labels on the explorer item (aria-expanded). However VoiceOver does not read it. This is an upstream bug and I have reached out to our VoiceOver contact.

**Comment by isidorn** [comment]
Our friends from Apple confirmed that they acknowledge the bug and are looking into a fix on the VoiceOver side.

**Comment by TheQuinbox** [comment]
Just tried in Version: 1.75.0-insider (Universal)
Commit: f076d553210e9ea65e7bccbfa2f7a786c10fcab2
Date: 2022-12-09T09:19:50.343Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.2.0
Sandboxed: Yes
And it seems fixed.

**Comment by isidorn** [comment]
Thanks for letting us know. So let me go ahead and close this.
If you see it again do let us know and we can reopen.
